src/components/EditDesignAndImgModal.js

```diff
@@ -44,12 +48,44 @@ const EditDesignAndImgModal = ({ size, title, handleEdit, theme, data }) => {
   };
 
   const onEdit = async () => {
+    console.log("update");
+
+    const imagesBaseUrl = `https://comy-test.s3.ap-northeast-1.amazonaws.com/users/${data.userId}`
```

Comment:
@maheralayoubi 
@Rezkaudi 

It is better to set this URL in **.env**, so it changes dynamically between the **Stage** and **Production** deployments.

_https://comy-test.s3.ap-northeast-1.amazonaws.com/users_

So this URL will be changed once we use the AWS **S3** account of **COMY**